pywbem_mock/config.py

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
This new module should be included in the docs.

Reply:
DONE